pkg/image/testdata/full-valid-example.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: 1.0
+arch: x86_64
```

Comment:
My expectation was that this would fall under the `image` section since it's dictating the type of image that's being created/modified. In other words, it feels like it's at the same level the user would be thinking "Raw or ISO?". Is there a reason you put it outside of that?

Reply:
Fair point. I actually misread the EIB plan document. Will move it under `image`.